components/game/copyButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+export const CopyButton = (props: {
+  textToCopy: string;
+  buttonText: string;
+}) => {
+  const handleClick = () => {
+    if ("clipboard" in navigator) {
+      navigator.clipboard.writeText(props.textToCopy);
+    } else {
+      return document.execCommand("copy", true, props.textToCopy);
```

Comment:
seems to be deprecated https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand

Reply:
yes, You're right I was just trying to increase the compatibility with browsers but it seems to be insecure and deprecated... I'll remove it